subs2srs.lua

```diff
@@ -342,13 +351,13 @@ local function export_to_anki(gui)
         sub['text'] = string.format("mpvacious wasn't able to grab subtitles (%s)", os.time())
     end
     local snapshot_timestamp = mp.get_property_number("time-pos", 0)
-    local snapshot_filename = filename_factory.make_snapshot_filename(snapshot_timestamp, config.snapshot_extension)
-    local audio_filename = filename_factory.make_audio_filename(sub['start'], sub['end'], config.audio_extension)
+    local video_filename = encoder.video.make_filename(sub['start'], sub['end'], snapshot_timestamp)
+    local audio_filename = encoder.audio.make_filename(sub['start'], sub['end'])
 
```

Comment:
I don't see why you'd need to move filename_factory to a different module since this doesn't have anything to do with the proposed feature.